core/node/da_clients/Cargo.toml

```diff
@@ -55,3 +55,14 @@ pbjson-types.workspace = true
 
 # Eigen dependencies
 tokio-stream.workspace = true
+rlp.workspace = true
+kzgpad-rs = { git = "https://github.com/Layr-Labs/kzgpad-rs.git", tag = "v0.1.0" }
```

Comment:
we only use crates.io -based dependencies, otherwise we won't be able to publish our crates